packages/@sparrow-workspaces/src/features/socket-explorer/layout/SocketExplorer.svelte

```diff
@@ -195,6 +195,7 @@
                         <div class="h-100 d-flex flex-column">
                           <ResponsePreviewNavigator
                             {webSocket}
+                            isWebApp={true}
```

Comment:
don't pass this isWebApp = true directly from here, take this from pages and pass here.

Reply:
ok